validator/client/validator.go

```diff
@@ -170,6 +267,7 @@ func (v *validator) WaitForKeymanagerInitialization(ctx context.Context) error {
 		return errors.New("key manager not set")
 	}
 	recheckKeys(ctx, v.db, v.km)
+	v.accountChangesSub = v.km.SubscribeAccountChanges(v.accountsChangedChannel)
```

Comment:
Just checking: Is it OK that we are doing this multiple times? Because this can be called once for every `Init` call throughout the validator's lifetime. Isn't there a memory leak somewhere?

Reply:
let me think about that , maybe I need to call validator done before initialize

Reply:
I think it is